torch/tensor.py

```diff
@@ -154,7 +160,7 @@ def __setstate__(self, state):
             raise RuntimeError('__setstate__ can be only called on leaf Tensors')
         if len(state) == 4:
             # legacy serialization of Tensor
-            self.set_(*state)
+            self.set_(*state)  
```

Comment:
nit: try to avoid unnecessary trailing space

Reply:
You can configure your editor to tell you about this. On vim I use `set listchars=tab:>-,trail:·`